tests/acs2/PMarkTest.py

```diff
@@ -101,9 +100,9 @@ def test_should_get_differences_2(self):
 
             # Then
             self.assertIsNotNone(diff)
-            self.assertEqual(c.CLASSIFIER_WILDCARD, diff[0])
-            self.assertEqual(c.CLASSIFIER_WILDCARD, diff[1])
-            self.assertEqual(c.CLASSIFIER_WILDCARD, diff[2])
+            self.assertEqual('#', diff[0])
```

Comment:
Shouldn't that be taken also from `config` class?

Reply:
Yep, sure.

Reply:
Actually, I do not think so: this is a test (didn't pay attention first) and the goal is to make it more readable. To my understanding in tests we should use literals as much as possible to reduce the influence of other code on test. 
Imagine there is a bug in ACS2Classification, or we used wrong configuration object in test initialization/assertion - and test failure/pass will not actually test what we intended to test.